ddp-workspace/projects/ddp-dsm-ui/src/app/sequencing-order/sequencing-order.component.ts

```diff
@@ -62,7 +62,7 @@ export class SequencingOrderComponent {
       },
       error: err => {
         if (err._body === Auth.AUTHENTICATION_ERROR) {
-          this.auth.sessionLogout();
+          this.auth.doLogout();
```

Comment:
Do we have documentation and/or tests on the backend that will help everyone understand how critical `Auth.AUTHENTICATION_ERROR` is?  Just in case we forget.

Can we also add a few lines in the DSM regression test doc that describe how to dial down the session timeout time and test a handful of the logout scenarios?  I don't think we need an automated test right now, but having a manual test documented would be a great start.

Reply:
Agree with your thinking Zim.
I'll work to do a few things here:

-  I was thinking about the same thing regarding the importance of the Auth.AUTHENTICATION_ERROR and briefly scanned our backend code to see if this was under test.  I didn't find it there.  Would it be okay to add a ticket so we add a backend unit or integration test that verifies this case?
- Get the token expiration time adjustments documented for dev, test and staging (I think we observed they're different at least between dev and test)
- Add steps to the multi-tab section on the regression spreadsheet for logout and token expiration.